Bikes/Bikes.AppHost/Program.cs

```diff
@@ -0,0 +1,17 @@
+﻿var builder = DistributedApplication.CreateBuilder(args);
+
+var kafka = builder.AddKafka("kafka")
+    .WithKafkaUI()
+    .WithDataVolume();
+
+var mongodb = builder.AddMongoDB("mongodb")
+    .WithDataVolume();
+
+var api = builder.AddProject<Projects.Bikes_Api_Host>("bikes-api")
+    .WithReference(mongodb)
+    .WithReference(kafka);
+
+var generator = builder.AddProject<Projects.Bikes_Generator>("bikes-generator")
+    .WithReference(kafka);
```

Comment:
Тоже добавьте ожидание.
```suggestion
_ = builder.AddProject<Projects.Bikes_Generator>("bikes-generator")
    .WithReference(kafka)
    .WaitFor(kafka);
```